src/main/java/com/yoheimuta/intellij/plugin/protolint/ProtolintAnnotator.java

```diff
@@ -47,15 +47,15 @@ public void apply(@NotNull PsiFile file, List<ProtolintWarning> warnings, @NotNu
         warnings.forEach(warning -> {
             int line = warning.getLine()-1;
             int endOffset = document.getLineEndOffset(line);
-            int startOffset = StringUtil.lineColToOffset(file.getText(), line, warning.getColumn());
+            int startOffset = StringUtil.lineColToOffset(file.getText(), line, warning.getColumn())-1;
```

Comment:
I am curious to know what the -1 is needed for here?

Reply:
I believe it's an issue with lines and columns starting from 0 instead of 1